locales/en/messages.json

```diff
@@ -2755,11 +2755,11 @@
     "dataflashButtonSaveFile": {
         "message": "Save flash to file..."
     },
-    "dataflashButtonSaveFileDeprecated": {
```

Comment:
This should stay and say `Save flash to file... (unsupported)` - not all users read the fine print.

Reply:
Changed as requested! Thanks.

Reply:
I prefer the button to be accurate about the status of the functionality, and tell users that they should not be using it, which is the official Betaflight position on this. I think `(unsupported)` is not too long to go on there.

Reply:
Mike, that is *your personal* opinion.
This is a community group but there is a time when you should listen to the people who use this feature day in day out.
There is no evidence that users should not be using `save flash to file` at this point in time.  Just none. 
It is simply not as unreliable as you make out.  The code may be dubious workarounds for MSP inadequacies, but it works really well.
Show me the issues raised about this on GitHub that you speak of.  I really couldn't find any - apart from some from a period in 2017-18 where there was a definite bug in some older 3.x releases.
After that - no complaints.
And it is not due to the 'deprecated' warning.  
It's because actually it works, and works reliably, if slowly.
I can see that you are adamant about this, but I feel equally strongly that you should consider that your perception may be biased on a handful of reports dating some time back.  These do not reflect the reality of the user experience over the last few years.  
When a person like me has not one failure in way more than 1000 log saves, it simply can't be as unreliable as you think.
I see no value in falsely over-emphasising the negatives of using save to file.  
Please let's just see how many reports we get if we remove the `deprecated` or `unsupported` text from the button.  We can always put it back if needed.

Reply:
I have checked over the logs I receive from people to check, and those on the Facebook betaflight log analysis group.  More than 50% - something like ⅔ - were saved via 'save to file'.  With such a massive number of people using it, we (or I) would be getting lots of reports - deprecated message notwithstanding - of problems.  We simply are not getting those kinds of reports.
People generally use Mass Storage if they have SD cards and record log after log.
Most flash users just use 'save flash to file', and do so without problems.